webapp/utils/web_client.jsx

```diff
@@ -21,18 +21,6 @@ class WebClientClass extends Client {
         this.setTeamId(TeamStore.getCurrentId());
     }
 
-    track = (category, action, label, property, value) => {
```

Comment:
How does this work with stuff like https://github.com/mattermost/mattermost-driver-javascript/blob/master/client.jsx#L175 ?